src/teamplay.c

```diff
@@ -199,6 +199,25 @@ void TeamplayEventItemTaken(gedict_t *client, gedict_t *item)
 	{
 		client->tp.took.item = (item->healamount >= 100 ? it_mh : it_health);
 	}
+	else if (streq(item->classname, "rune"))
+	{
+		if (item->ctf_flag & CTF_RUNE_RES)
+		{
+			client->tp.took.item = it_rune1;
+		}
+		else if (item->ctf_flag & CTF_RUNE_STR)
+		{
+			client->tp.took.item = it_rune2;
+		}
+		else if (item->ctf_flag & CTF_RUNE_HST)
+		{
+			client->tp.took.item = it_rune3;
+		}
+		else if (item->ctf_flag & CTF_RUNE_RGN)
+		{
+			client->tp.took.item = it_rune4;
```

Comment:
Guess it overlaps with bots, but could be valuable before bots? How does this relate to `//ktx took` messages to client/mvd, would be pretty nice to get rid of the nasty hack in demo playback / qtv for tracking runes.

Reply:
Also, why not check for classname item_flag_team1 and item_flag_team2 here?